rho/postprocessing.py

```diff
@@ -229,9 +235,11 @@ def empty_output_message(val, name):
     if JBOSS_EAP_INSTALLED_VERSIONS in fact_names:
         val[JBOSS_EAP_INSTALLED_VERSIONS] = (
             empty_output_message('; '.join(jboss_releases), 'jboss'))
+        val[JBOSS_EAP_INSTALLED_VERSIONS + MR] = jboss_releases
```

Comment:
I think we still want to display the installed versions fact in the report.

Reply:
The user-readable version of this is `'; '.join(jboss_releases)` but with a nicer error message when we have no releases. Do you specifically want the raw list with none of the extra error message stuff? (I think we have to string-format it for display regardless, it's just a question of how we do that.)

Reply:
I did some research. According to the csv module documentation (https://docs.python.org/2/library/csv.html#writer-objects), the input to `DictWriter.writerow()` must be a dict whose values are strings and numbers. So list structures like this will have to be filtered out before the CSV output step.

I don't think the `-mr` thing is the best approach here, happy to talk about others. As I said, I did it mostly for speed.